### PR TITLE
Updated README.md with basic tutorial.

### PR DESCRIPTION
**Pull request by Pyrodox**
for more clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
+- [Visual Studio Code](https://code.visualstudio.com/download)
+- [Boost](https://www.boost.org/users/download/)
+- [CMake](https://cmake.org/download/) (binary, either of the x64 if on Windows)
 
-However, before you watch the video, you must set up some of the larger software.
-For Qt, check these options
+## Pre-Setup Instructions
 
-For Visual Studio Community, check Desktop development with C++, then these options
+### Qt SDK Options
```
Comment (Pyrodox) [inline README.md:19]: image
image

File: README.md [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
 
-Note: Do NOT check SDK 10.0.19041.0 as you are potentially subject to a very frustrating bug
+[insert image]
 
-For Visual Studio Build Tools, check Desktop development with C++, then these options
+### Visual Studio Installation Options
+Check `Desktop Development with C++` with these options.
```
Comment (Pyrodox) [inline README.md:24]: image

File: README.md [hunk 3]
```diff
@@ -1,40 +1,56 @@
...
+***Do NOT check SDK 10.0.19041.0 due to a bug***
 
-For environment variables, make sure you have paths to VScode bin, CMake bin, and your compiler's bin (MinGW for example)
+[insert image]
```
Comment (Pyrodox) [inline README.md:27]: Example:
image

File: README.md [hunk 4]
```diff
@@ -1,40 +1,56 @@
...
 
-VScode extensions: CMake Tools, C/C++, CMake, CMake Language Support
+### Visual Studio Build Tools Installation Options
+Check `Desktop Development with C++` with these options.
+
+[insert image]
```
Comment (Pyrodox) [inline README.md:32]: image

**Review by Pyrodox — COMMENTED** [review]
Images to be added

**Review by richardwei6 — APPROVED** [review]

**Pull request merged**